### DISCUSSION TITLE
Flagged as trojan by antivirus

### DISCUSSION THREAD

**Discussion by jmacd867** [akiver/cs-demo-manager#889] [Q&A] [answered]
Hey I keep getting a windows defender notification from CS Demo Manager and when I upload it to virus total it comes back with a virus Trojan.WinGo.Agent. Any ideas?

**Comment by akiver** [comment, marked as answer]
Same as https://github.com/akiver/cs-demo-analyzer/issues/4 - getting a code signing certificate on Windows is expensive, that's why the app is not signed and may be reported by anti-virus.

As long as you download the installer from GitHub you are safe - the binaries are built and uploaded from a GitHub action.
You could also [build the app](https://cs-demo-manager.com/docs/development/building) from the sources if you want to.
